core/tests/test_utils.py

```diff
@@ -835,37 +815,30 @@ def _get_response(
             webtest.TestResponse. The test response.
         """
         if params is not None:
-            self.assertTrue(isinstance(params, dict))
+            self.assertIsInstance(params, dict)
 
-        expect_errors = False
-        if expected_status_int >= 400:
-            expect_errors = True
+        expect_errors = expected_status_int >= 400
```

Comment:
Do we even need this variable? We check for the exact `expected_status_int` number anyway. Ditto in other places.